RInAppMessaging/Classes/RInAppMessaging.swift

```diff
@@ -11,9 +11,11 @@
 /// Conforms to NSObject and exposed with objc tag to make it work with Obj-c projects.
 @objc public class RInAppMessaging: NSObject {
 
-    private static let inAppQueue = DispatchQueue(label: "IAM.Main", qos: .utility, attributes: [])
     internal private(set) static var initializedModule: InAppMessagingModule?
     private(set) static var dependencyManager: DependencyManager?
+    private static var inAppQueue: DispatchQueue? {
+        dependencyManager?.resolve(type: DispatchQueue.self)
```

Comment:
Would it be useful to have a guard here to catch this being nil and log an error?